### PR TITLE
doc updates for kustomize changes 

### PR DESCRIPTION
**Pull request by jsanda**
This PR includes a handful of doc updates to the changes made in #168.

### CODE REVIEW

File: test/e2e/suite_test.go [hunk 1]
```diff
@@ -113,7 +113,7 @@ func e2eTest(ctx context.Context, fixture TestFixture, deployTraefik bool, test
 		}
 
 		err = beforeTest(t, namespace, fixtureDir, f, deployTraefik)
-		defer afterTest(t, namespace, f, deployTraefik)
+		//defer afterTest(t, namespace, f, deployTraefik)
```
Comment (burmanm) [inline test/e2e/suite_test.go:116]: Unintentional comment out?

Comment (jsanda) [reply test/e2e/suite_test.go:116]: Yeah, thanks for spotting

**Review by burmanm — APPROVED** [review]

**Comment by jsanda** [comment]
There are some more Makefile updates I need to make since the operator is deployed to the `k8ssandra-operator` namespace instead of `default` now.

**Review by adejanovski — APPROVED** [review]
Everything works as expected ✅

**Pull request merged**